### DISCUSSION TITLE
Grbl 1.1, laser engraver, end stops

### DISCUSSION THREAD

**Discussion by wfishrn** [arkypita/LaserGRBL#925] [Q&A]
Sorry if I seem a bit dumb or if im asking a question that has been asked multiple times, Im new to all of this. After a year of learning Fusion 360, and how to 3D print to make the parts needed for my engraver I have completed the build and am now starting to tune and run the system. 
  End stops not working at all, they are enabled in LaserGRBL, they are the makerbot 3 wire style, When I run a homing cycle it shows "Alarm: Homing fail" next line Grbl 0.9j. If i activate the switches manually I get no response.  
  End stop plugged into Y+ end stop, red wire to white and Black wire to Black, Green wire ????? so where does this signal wire go???
Thank You Much
W Fish RN

**Comment by gmmanonymus111** [comment]
[Here](https://github.com/gnea/grbl/wiki/Wiring-Limit-Switches)'s a detailed description on limit switches for GRBL
